tasks/elex.js

```diff
@@ -22,12 +22,25 @@ module.exports = function (grunt) {
     const offline = grunt.option("offline");
     const zero = grunt.option("zero");
 
+    const countyLevelDataForStates =
+      "AL,AK,AZ,AR,CA,CO,DC,DE, FL,GA,HI,ID,IL,IN,IA,KS,KY,LA,MD,MI,MN,MS,MO,MT,NE,NV,NJ,NM,NY,NC,ND,OH,OK,OR,PA,SC,SD,TN,TX,UT,VA,WA,WV,WI,WY";
+
+    const townshipLevelDataForStates = "CT,ME,MA,NH,RI,VT";
     const tickets = [
       {
         date: "2024-11-05",
         params: {
           officeID: "G,S,P",
           level: "FIPSCode",
+          statePostal: countyLevelDataForStates,
+        },
+      },
+      {
+        date: "2024-11-05",
+        params: {
+          officeID: "G,S,P",
+          level: "ru",
```

Comment:
I thought the `ru` level would get a mix of township and county-level data, so you don't need to make multiple requests. That's not working?